boards/arm/96b_carbon/96b_carbon.dts

```diff
@@ -41,6 +41,16 @@
 	clock-frequency = <I2C_BITRATE_FAST>;
 };
 
+&spi1 {
+	interrupts = <35 5>;
```

Comment:
I think you can / should delete the interrupts properties from the board files. You've already defined them at SOC level, which is where they belong.

Reply:
Done.